src/main/java/core/basesyntax/service/operations/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        //storage = (Storage) Storage.fruits;
+        StringBuilder builder = new StringBuilder();
+        builder.append("fruit,quantity")
+                .append(System.lineSeparator());
+        for (String name: Storage.fruits.keySet()) {
+            String key = name.toString();
+            String value = Storage.fruits.get(name).toString();
+            builder.append(key + "," + value)

```

Comment:
Instead of concatenating strings using '+', you should use the append() method of the StringBuilder for better performance.

Reply:
ok, done